mediated_transfer.rst

```diff
@@ -0,0 +1,71 @@
+Mediated transfers
+##################
+
+Overview
+========
+
+The protocol supports mediated transfers. Nodes may use them to send payments.
+A :term:`Mediated transfer` may be cancelled and can expire until the initiator reveals the secret.
```

Comment:
can you move this sentence below? I would expect first the explanation about what a mediated transfer is and then how it can fail